testutil/promrated/go.mod

```diff
@@ -0,0 +1,137 @@
+module github.com/obolnetwork/charon/promrated
+
+go 1.19
```

Comment:
I'd rather not introduce another gomod, so use the repo root as docker context but point to dockerfile in other folder.